pdns/sha.hh

```diff
@@ -59,7 +59,11 @@ class SHADigest
 {
 public:
   SHADigest(unsigned int bits) :
+#if OPENSSL_VERSION_NUMBER < 0x10100000L
```

Comment:
We have taken a different approach in digests.hh, since our configure checks whether `EVP_MD_CTX_new` and `EVP_MD_CTX_free` actually exist.

Reply:
Right, I was looking into the libssl configure check, while it should have been the libcrypto one